src/main/java/org/sonarqube/gradle/SonarQubeTask.java

```diff
@@ -38,6 +41,33 @@
  */
 public class SonarQubeTask extends DefaultTask {
 
+  private static final Logger LOGGER = Logging.getLogger(SonarQubeTask.class);
+
+  public static final LogOutput LOG_OUTPUT = new LogOutput() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce this anonymous class number of lines from 24 to at most 20, or make it a named class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1188)